File: 167-two-sum-ii-input-array-is-sorted/167-two-sum-ii-input-array-is-sorted.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int[] m = new int[2];
        int l = 0;
        int r = numbers.length - 1;
        while(l<r){
            if((numbers[l] + numbers[r]) > target){
                r --;
            }
            else if((numbers[l] + numbers[r]) < target){
                l++;
            }
            else{
                m[0] = l+1;
                m[1] = r+1;
                l++;
                r--;
            }
        }
       return m; 
    }
}

File: 167-two-sum-ii-input-array-is-sorted/167-two-sum-ii-input-array-is-sorted.py
class Solution(object):
    def twoSum(self, numbers, target):
        """
        :type numbers: List[int]
        :type target: int
        :rtype: List[int]
        """
        m = []
        l = 0
        r = len(numbers) - 1
        
        while(l<r):
            if((numbers[l] + numbers[r]) > target):
                r -= 1
            elif((numbers[l] + numbers[r]) < target):
                l += 1
            else:
                m.append(l+1)
                m.append(r+1)
                return m
        
                    
    
                    